content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,85 @@
-click on filter and check Won/Lost.
-
-.. image:: win_loss/win_loss01.png
+.. image:: win_loss/reporting-tab-and-pipeline-view.png
     :align: center
+    :alt: View of CRM page showing the Reporting tab with the Pipeline view highlighted.
 
-You can also change the *Measures* to *Total Revenue*.
+Open the *Search Panel* dropdown, and make the following selections:
 
-.. image:: win_loss/win_loss02.png
-    :align: center
+#. Filters: *Won*.
+#. Filters: *Lost*.
+
+   - Optional: Removing or adjusting the *Created On* filter to reflect the last 30 days, the last
+     quarter. By default, it reflects the entire year.
 
-You also have the ability to switch to a pie chart view.
+   - Optional: Show only leads that have reached the last stage of the sales pipeline (Closed) by
+     adding a custom filter for *Last Stage Update is >= DATE*.
+#. Group By: *Salesperson*.
+#. Group By: *Add Custom Group --> Active*.
 
-.. image:: win_loss/win_loss03.png
+   - Optional: Group by *Sales Team* or *Company* instead of by "Salesperson*.
+
+.. image:: win_loss/search-panel-filters-and-group-by-options.png
     :align: center
+    :alt: View of CRM page showing the Search Panel open and the Won and Lost filters highlighted.
+
+The *Search Panel* should read: *Filter: Created On: YYYY, Filter: Won or Lost, Group By:
+Salesperson > Active*. If so, it now displays the *count* of wins and losses for each salesperson.
+Hover over the chart to see the percentage of wins (true) or losses (false). Click on a
+section of the chart to open a list of those accounts.
+
+.. image:: win_loss/win-loss-ratio-bar-chart.png
+   :align: center
+   :alt: View of Win/Loss Ratio report as a bar chart.
+
+Additional Measurements and Views
+=================================
+By default, the Win/Loss report measures the total *count* of leads. Click the
+*Measures* button to select other options such as *Expected Revenue*.
+
+Click the chart icons along the top to switch the graph type from a Bar Chart to a Line Chart or Pie
+Chart.
+
+Additionally, the report can be changed from a Graph view to a Pivot view, Cohort view, or
+List view.
+
+.. image:: win_loss/measure-button-menu-pie-chart-view-and-graph-type-button.png
+   :align: center
+   :alt: The Measures dropdown menu and pie chart view and pivot view.
+
+Add a Win/Loss Report to Dashboards
+===================================
+After configuring the views and measurements, a Win/Loss report can be saved to a spreadsheet and/or
+dashboard for easier viewing in the future.
+
+From the Win/Loss report, switch to the *Pivot* view. Click the *Measures* button
+and select *count*. Click *Insert in Spreadsheet*.
+
+.. image:: win_loss/select-count-and-insert-in-spreadsheet.png
+   :align: center
+   :alt: Win/Loss report in pivot view showing the Measures dropdown menu.
+
+Select *Blank Spreadsheet* and click *Confirm*. Name the new spreadsheet,
+and add charts or additional formulas as desired.
```

Comment:
Have you ever tried the command `make review`? it helps catch instances where you can fit a few more words onto the line and get it as close to 100 characters as possible. 

To run make review, go to github and copy the file path, in this case
`content/applications/sales/crm/performance/win_loss.rst`
then, REMOVE content/, so u are left with
`applications/sales/crm/performance/win_loss.rst`

Go to terminal, and in the documentation folder, run `make review`
and then, enter the path we just made
When they ask the question about 100 char, just press the Enter button on your keyboard
![image](https://github.com/odoo/documentation/assets/23611711/8d760867-d397-4bef-a5e9-a53166d1c878)

Then, you'll see messages like this:
![image](https://github.com/odoo/documentation/assets/23611711/cb8c36c0-2ed3-45c9-ade9-205872db4fa3)

Saying that in lines 36, 45, 51, 63, 70, and 73, u have an early line break and u can reformat. (For example, shove more words from line 71 onto 70) 
```suggestion
Select *Blank Spreadsheet* and click *Confirm*. Name the new spreadsheet, and add charts or
additional formulas as desired.
```